webapp/README.md

```diff
@@ -0,0 +1,28 @@
+# Mattermost Web App
+
+This folder contains the client code for the Mattermost web app. It's broken up into multiple packages each of which either contains an area of the app (such as `playbooks` or `boards`) or shared logic used across other packages (such as the packages located in the `platform` directory). For anyone who's used to working in [the mattermost/mattermost-webapp repo](https://github.com/mattermost/mattermost-webapp), most of that is now located in `channels`.
+
+## npm Workspaces
+
+To interact with a workspace using npm, such as to add a dependency or run a script, use the `--workspace` (or `--workspaces`) flag. This can be done when using built-in npm commands such as `npm add` or when running scripts. These should be done from this folder without cding into any of the package directories.
```

Comment:
For my own curiosity I Googled if `cding` is a common abbreviation for changing directory.  The first page is mostly about cross-dressing.  `cd'ing` is apparently about taking creepy candid pics of people.  `cd-ing` seems to get some relevant hits.  Or perhaps just write it out.

Reply:
Oh weird. Yeah, I'll change that. I spent enough time trying to figure out if `cd` should be lower case or upper case there already, and I still wasn't happy with how it looked